runners/flink/src/test/java/org/apache/beam/runners/flink/PortableTimersExecutionTest.java

```diff
@@ -185,11 +178,10 @@ public void processElement(ProcessContext context) {
             Collections.emptyList());
 
     jobInvocation.start();
-    long timeout = System.currentTimeMillis() + 60 * 1000;
+    long timeout = System.currentTimeMillis() + 2 * 60 * 1000;
     while (jobInvocation.getState() != Enum.DONE && System.currentTimeMillis() < timeout) {
```

Comment:
To further improve this test: We can remove the timeout related logic here. The test already has a timeout set above.

Reply:
Done